test/unit/mock/cmd/bash-c-type-nginx

```diff
@@ -0,0 +1 @@
+nginx is /usr/sbin/nginx
```

Comment:
Hey adam I am bit confused...are suggesting a test for the  command('nginx').exists?